File: C++/1066.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    int a;
    int pos=0;
    int neg=0;
    int imp=0;
    int par=0;
    
    for(int i=0;i< 5;i++){
        cin >> a;
        
        a%2 == 0 ||a == 0 ? par++ : imp++;
        if(a>0)pos++;
        if(a<0)neg++;
    }
    cout << par << " valor(es) par(es)" << endl;
    cout << imp << " valor(es) impar(es)" << endl;
    cout << pos << " valor(es) positivo(s)" << endl;
    cout << neg << " valor(es) negativo(s)" << endl;

    return 0;
}

File: Java/1066.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {

    public static void main(String[] args) throws IOException {
        
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        
        int a, pos=0, neg=0, imp=0, par=0;
        
        for(int i=0;i< 5;i++){
            a = Integer.parseInt(in.readLine());
            
            if(a % 2 == 0 ||a == 0) {
                par++;
            }else {
                imp++;
            }
            if(a>0){pos++;}
            if(a<0){neg++;}
        }
        
        System.out.println(par + " valor(es) par(es)");
        System.out.println(imp + " valor(es) impar(es)");
        System.out.println(pos + " valor(es) positivo(s)");
        System.out.println(neg + " valor(es) negativo(s)");
    }

}